pkg/config/components/honeycombExporter.yaml

```diff
@@ -0,0 +1,35 @@
+name: ToHoneycomb
+kind: HoneycombExporter
+summary: Export traces to Honeycomb in Honeycomb's event format
```

Comment:
```suggestion
summary: Export telemetry to Honeycomb
```

Reply:
my suggestion here is that this component could be generic enough to emit any kind of data to honeycomb, the export format might not be necessary to call out in the documentation of the component.

Reply:
This component today only applies to Refinery, but it's a good point that logs can also flow through Refinery. So I changed it to telemetry but left in the "event format" part.